Crisan Cristian Alexandru/07 3.8/main.c

```diff
@@ -0,0 +1,66 @@
+#include <stdio.h>
+#include <stdlib.h>
+/*3.8. Write a function to compute the product of two polynomials,
+ using only pointers and pointer expressions. */
+void PolynomRead(int *Polynom,int PolGr)
+{
+    for(int i=PolGr;i>=0;i--)
+    { printf(" The coefficient of x^%d is:",i);
```

Comment:
Ugly formatted code, please properly format

Reply:
What exactly is ugly? The names of the variables, or the layout of the code?

Reply:
The layout of the code! 
Variables seem to be consistently named
Pretty please format code